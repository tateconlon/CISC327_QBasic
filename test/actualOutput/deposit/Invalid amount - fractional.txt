login as agent successful
Account 1234567 does not exist
transaction code 1.99 invalid
Transaction summary written and system logged out successfully