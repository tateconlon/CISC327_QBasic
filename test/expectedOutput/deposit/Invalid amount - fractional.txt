login as agent successful
1.54 is not a valid number 
Transaction summary written and system logged out successfully